pkg/web/router.go

```diff
@@ -62,35 +65,44 @@ func NewRouter(idpConfig *idp.Config, schemasConfig *schemas.Config, rulesConfig
 	identities.NewAPI(
 		identities.NewService(externalConfig.KratosAdmin().IdentityApi(), tracer, monitor, logger),
 		logger,
-	).RegisterEndpoints(router)
+	).RegisterEndpoints(router).
+		SetupValidation(vldtr)
 	clients.NewAPI(
 		clients.NewService(externalConfig.HydraAdmin(), tracer, monitor, logger),
 		logger,
-	).RegisterEndpoints(router)
+	).RegisterEndpoints(router).
+		SetupValidation(vldtr)
 	idp.NewAPI(
 		idp.NewService(idpConfig, tracer, monitor, logger),
 		logger,
-	).RegisterEndpoints(router)
+	).RegisterEndpoints(router).
+		SetupValidation(vldtr)
 	schemas.NewAPI(
 		schemas.NewService(schemasConfig, tracer, monitor, logger),
 		logger,
-	).RegisterEndpoints(router)
+	).RegisterEndpoints(router).
+		SetupValidation(vldtr)
 	rules.NewAPI(
 		rules.NewService(rulesConfig, tracer, monitor, logger),
 		logger,
-	).RegisterEndpoints(router)
+	).RegisterEndpoints(router).
+		SetupValidation(vldtr)
 	roles.NewAPI(
 		roles.NewService(externalConfig.OpenFGA(), wpool, tracer, monitor, logger),
 		tracer,
 		monitor,
 		logger,
-	).RegisterEndpoints(router)
+	).RegisterEndpoints(router).
+		SetupValidation(vldtr)
 	groups.NewAPI(
 		groups.NewService(externalConfig.OpenFGA(), wpool, tracer, monitor, logger),
 		tracer,
 		monitor,
 		logger,
-	).RegisterEndpoints(router)
+	).RegisterEndpoints(router).
+		SetupValidation(vldtr)
+
+	router = router.With(vldtr.Middleware).(*chi.Mux)
```

Comment:
can we bundle this up with line 65 where authorization middleware is plugged in?